zerver/transaction_tests/test_user_groups.py

```diff
@@ -64,7 +64,7 @@ def wait_after_recursive_query(*args: Any, **kwargs: Any) -> UserGroup:
 
 
 class UserGroupRaceConditionTestCase(ZulipTransactionTestCase):
-    created_user_groups: list[UserGroup] = []
+    created_user_groups: list[NamedUserGroup] = []
```

Comment:
Needed to change this type since `can_manage_group` only exists on NamedUserGroup type + all the groups in question are NamedUserGroups